composer.json

```diff
@@ -29,24 +32,23 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.4 || ~8.0.0",
         "laminas-api-tools/api-tools": "^1.3",
         "laminas-api-tools/api-tools-rest": "^1.3.2",
-        "laminas/laminas-hydrator": "^2.4.2 || ^3.0",
+        "laminas/laminas-hydrator": "^3.0 || ^4.0",
```

Comment:
Let's just use `^4.0` here: supporting both versions, while nice, is incompatible with our ability to keeps things maintained.

In practice, supporting multiple majors **can** be done, but not without introducing dangerous and unnecessary long-term fragility.